mucommander-core/src/main/java/com/mucommander/search/SearchProperty.java

```diff
@@ -43,6 +43,8 @@ public enum SearchProperty {
     SEARCH_TEXT("text", "search_dialog.search_text", ""),
     TEXT_WHOLE_WORDS("whole-words", "used-via-find-dialog1", Boolean.toString(false)),
     TEXT_SEARCH_FORWARD("forward", "used-via-find-dialog2", Boolean.toString(true)),
+    OPEN_WITH_FOUND_SELECTED(null,"not-used-in-ui", Boolean.toString(false)),
```

Comment:
I don't think that this is the best place to store it - it's not really a search property, is it?
did you try to inject it all the way to TextEditorImpl?

Reply:
Didn't want to mess 'all the way' to TextEditorImpl - of course I had the same doubts as you, however, yes, it is not search property, but still conveys search related information :)

Reply:
PR updated, @ahadas pls check if it is OK now.